app/javascript/packages/address-search/components/full-address-search-input.tsx

```diff
@@ -1,34 +1,22 @@
 import { TextInput, SelectInput } from '@18f/identity-components';
-import type { FormattedLocation, LocationQuery } from '@18f/identity-address-search/types';
-import type { RegisterFieldCallback } from '@18f/identity-form-steps';
 import { useDidUpdateEffect } from '@18f/identity-react-hooks';
 import { SpinnerButtonRefHandle, SpinnerButton } from '@18f/identity-spinner-button';
 import { ValidatedField } from '@18f/identity-validated-field';
 import { useI18n } from '@18f/identity-react-i18n';
 import { useCallback, useEffect, useRef, useState } from 'react';
 import useValidatedUspsLocations from '../hooks/use-validated-usps-locations';
-
-interface FullAddressSearchInputProps {
-  disabled?: boolean;
-  locationsURL: string;
-  onError?: (error: Error | null) => void;
-  onFoundLocations?: (
-    address: LocationQuery | null,
-    locations: FormattedLocation[] | null | undefined,
-  ) => void;
-  onLoadingLocations?: (isLoading: boolean) => void;
-  registerField?: RegisterFieldCallback;
-  usStatesTerritories: string[][];
-}
+import type { FullAddressSearchInputProps } from '../types';
 
 export default function FullAddressSearchInput({
   disabled = false,
   locationsURL,
+  onContinue,
```

Comment:
From refinement:

If you set `onContinue` to an empty function (`() => undefined`), you should be able to get rid of the bang on line 77 and simplify the type def as well.

Reply:
TL;DR: I see the issue and it isn't worth resolving in this ticket.

---

Even if you do something like this:

```suggestion
  onContinue = async () => undefined,
```

Since `handleLocationSelect` could be `null` (though I can't see how...), it will override any default value to your constructor since it is a "real" value unlike `undefined`.

The only way to really fix it would be to ensure that `null` is not a return value for `handleLocationSelect`, modify the types (remove `null` from the declaration), and provide a default value to `onContinue`. The investigation is ensuring `null` is never passed in seems beyond the scope of work here!